dev.ts

```diff
@@ -75,13 +78,41 @@ export default async function dev(
   await import(entrypointHref);
 }
 
-async function generateDevManifestData(dir: string): Promise<DevManifestData> {
+async function generateDevManifestData(
+  dir: string,
+  imports: Record<string, any>,
+): Promise<DevManifestData> {
+  const mapWith = (pre: string) => (list: string[]) =>
+    list.map((name) => "./" + join(pre, name));
   const [manifestFile, sections, functions] = await Promise.all([
     collect(dir),
-    collectSections(dir),
-    collectFunctions(dir),
+    collectSections(dir).then(mapWith("./sections")),
+    collectFunctions(dir).then(mapWith("./functions")),
   ]);
-  const schemas = await extractAllSchemas(sections, functions);
+
+  let schemas = await extractAllSchemas(sections, functions);
+
+  // "imports" is of format { "nameOfImport" : manifest }
+  for (const [key, importManifest] of Object.entries(imports)) {
```

Comment:
have you took a look here at the workbench API? It's a sanity check
https://github.com/deco-cx/live/blob/main/utils/workbench.ts#L21